pkg/driver/controller.go

```diff
@@ -489,7 +484,8 @@ func validateControllerUnpublishVolumeRequest(req *csi.ControllerUnpublishVolume
 
 func (d *ControllerService) ControllerGetCapabilities(ctx context.Context, req *csi.ControllerGetCapabilitiesRequest) (*csi.ControllerGetCapabilitiesResponse, error) {
 	klog.V(4).InfoS("ControllerGetCapabilities: called", "args", req)
-	var caps []*csi.ControllerServiceCapability
+
+	caps := make([]*csi.ControllerServiceCapability, 0, len(controllerCaps))
```

Comment:
If we are changing cap to capability should we change caps to capabilities ?

Reply:
I was changing cap to capability to avoid colliding with built-in `cap` function, which this linter spotted.

Open to changing controllerCaps to controllerCapabilities if others want that in this PR.

Reply:
Honestly I think that `caps` is clear enough in the context (and especially in `controllerCaps` keeps the variable size within reason), but don't have a strong opinion either way.

Reply:
Sounds good, resolving then.